doc/source/Tutorials/GettingStarted/index.md

```diff
@@ -1,407 +1,499 @@
 
-The render should now be in the process of updating to show the results of our edit.
+Create the render settings nodes:
 
-Filtering a shader assignment
------------------------------
+1. Select the Group node.
 
-Everything is looking rather uniform right now, because so far we've used one material for everything. Let's assign a metallic material to a few objects to add some interest.
+2. Create a StandardOptions node (_Scene_ > _Globals_ > _StandardOptions_). It will automatically connect to the output of the Group node and become selected.
 
-- Make a Disney material as before (_/Appleseed/Shader/Material/As_Disney_Material_)
-- Edit it using the NodeEditor
-	- Set **Metallic** to `0.8`
-	- Set **Roughness** to `0.4`
-- Create a Shader assignment as before (_/Scene/Attributes/ShaderAssignment_), and make sure the
-  new material is connected into it.
-- Insert the new ShaderAssignment into the connection below the previous one.
+3. Create an AppleseedOptions node (_Appleseed_ > _Options_). It will connect automatically to the output of the StandardOptions node and become selected.
 
-![Second shader assignment](images/secondShaderAssignment.png)
+4. Create an Outputs node (_Scene_ > _Globals_ > _Outputs_). It will connect and become selected like the others.
 
-The viewer should update to show the new assignment.
+5. Create an InteractiveAppleseedRender node (_Appleseed_ > _InteractiveRender_).
 
-![Second shader assignment render](images/secondShaderAssignmentRender.png)
+6. Create a Catalogue node (_Image_ > _Utility_ > _Catalogue_). It does not need to be connected to anything. Instead, place it next to the InteractiveAppleseedRender node.
 
-Oops. Because the new ShaderAssignment is downstream from the first one, it has overridden the first assignment, and now _everything_ is chrome. We need a way of limiting the objects that the second ShaderAssignment is applied to - in Gaffer this is referred to as _filtering_ the ShaderAssignment, and it is done using special Filter nodes :
+    ![The render-related nodes](images/graphEditorRenderSettings.png "The render-related nodes")
 
-- Create a PathFilter node (_/Scene/Filters/PathFilter_). This chooses which objects to affect based on their names.
-- Use the NodeEditor to add `/group/GAFFERBOT/C_torso_GRP/C_head_GRP/C_head_CPT/L_ear001_REN` to the **Paths**. This is the
-  full name of the left ear of the robot, as you see it listed in the HierarchyView.
-- Connect the output of the PathFilter into the filter input on the right hand side of the ShaderAssignment1 node.
+Briefly, here is the function of each of these nodes:
+- StandardOptions node: Determines the camera, resolution, and blur settings of the scene.
+- AppleseedOptions node: Determines the settings of the Appleseed renderer.
+- Outputs node: Determines what kind of output render will be created.
```

Comment:
"Determines what kind of output _**images**_ will be created." ?

Reply:
this one is about images though. an Output scene node is specifying what kind of images the renderer should generate. And those results can only be viewed via an image node. So I don't know why we'd want to distinguish between 3d and 2d in this case, its the one node that overlaps the two.

Reply:
Hmm, then maybe I'll mention that unique overlap.